### PR TITLE
Edits to global-product-delivery.md

### PR DESCRIPTION
**Pull request by joeyobr**
Edited this topic in prep for featuring it on social media.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/6d22b133c3c37e1af6c3f1296dcbfab3d66b1f4d/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [88](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4a4cb96-ef60-40a8-a892-0ea6c621f5d2) | 100 | 100 | 73 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/methodology/global-product-delivery.md [hunk 1]
```diff
@@ -43,18 +38,18 @@ Consider these aspects, that may vary widely by locale and language:
...
-* People's names
-* Calendars
-* Date and time formats
-* Number formats
-* Currency unit and formatting of monetary values
-* Postal addresses
-* Telephone numbers
-* Measurement units
-* Paper sizes
+* Language and language variants, including word choice, spelling, and punctuation.
```
Comment (jowilco) [inline globalization/methodology/global-product-delivery.md:41]: I'm not sure about this change. Style Guide says:
Don’t use end punctuation in headlines, headings, subheadings, UI titles, UI text, **or simple lists (three or fewer words per item)**.
On average, we're lower than 3 words per item.

File: globalization/methodology/global-product-delivery.md [hunk 2]
```diff
@@ -94,10 +89,7 @@ See [Roles and responsibilities](roles-and-responsibilities.md)
 
 ## Start early
 
-When you build global awareness and localizability into your product and process as early as possible, you reduce risk and cost.
-
-In our experience, world-readiness can be quite challenging to retrofit to an existing application.
-For a pre-existing application, many components may need to be redesigned, particularly in the user interface.
+When you build global awareness and localizability into your product and process as early as possible, you reduce risk and cost. In our experience, world-readiness can be quite challenging to retrofit to an existing application. For a pre-existing application, many components may need to be redesigned, particularly in the user interface.
```
Comment (jowilco) [inline globalization/methodology/global-product-delivery.md:92]: Just FYI, this was a Paul anachronism - for some reason he felt that every sentence should be on its own line. I'm fine with removing the line breaks, and I did that elsewhere.

File: globalization/methodology/global-product-delivery.md [hunk 3]
```diff
@@ -110,20 +102,15 @@ When starting a project, it's vital to get these fundamentals in place as early
 
 2. Produce and test localized builds.
 
-   [Pseudolocalization](pseudolocalization.md) is an excellent strategy to enable build and test without the cost of translation.
-
-3. Validate the localization round-trip from development to translator and back.
+   [Pseudolocalization](pseudolocalization.md) is an excellent method to enable build and test without incurring the cost of translation.
 
+3. Validate the localization round-trip from development to translator and back.
```
Comment (jowilco) [inline globalization/methodology/global-product-delivery.md:107]: Note the lint issue with terminating spaces.

**Review by joeyobr — COMMENTED** [review]
I agree. Please remove the periods on the bullet points.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/dac2f6311ed10f3547bb64988734cde174cb7bc7/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [88](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89e8b03e-856a-474c-8b20-111187eb3d52) | 100 | 100 | 73 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/73aac5ebd0ef8d14152425eda005b4a274123143/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [88](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/82941e3e-e7a4-494b-baad-482369b64d3a) | 100 | 100 | 73 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/methodology/global-product-delivery.md](https://github.com/MicrosoftDocs/globalization/blob/bff9081038ccfa30695e17e4aaf33b09e5eecfb1/globalization%2Fmethodology%2Fglobal-product-delivery.md) | [88](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/20c8fee7-33b0-48ca-9026-4191b14ac706) | 100 | 100 | 73 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**